doc/nrf/includes/wifi_credentials_shell.txt

```diff
@@ -0,0 +1,21 @@
+Configuring Wi-Fi access point credentials
+==========================================
+
+This sample uses the :ref:`lib_wifi_credentials` library to manage Wi-Fi credentials.
+Before the sample can connect to a Wi-Fi network, you must configure at least one credential set.
+
+Once your device has been flashed with this sample, connect to your device's UART interface and add credentials using the following command:
+
+.. parsed-literal::
+   :class: highlight
+
+   wifi_cred add *NetworkSSID* WPA2-PSK *NetworkPassword*
```

Comment:
```suggestion
   wifi_cred add <"NetworkSSID"> <SecurityMode (WPA2-PSK, WPA3-SAE, ...)> <NetworkPassword>
```
Maybe find a way to reference the supported security types. Somehow, and instructions how to find it.

wifi scan can be used to locate available APs and their security mode so instructions on that would be nice.

Reply:
Using wifi scan means that we also need to include the wifi shell in our net samples.